### PR TITLE
Suggestion of text on CRS 

### PR DESCRIPTION
**Pull request by timrobertson100**
https://github.com/Leaflet/Leaflet/issues/5072

### CODE REVIEW

**Comment by IvanSanchez** [comment]
@timrobertson100 Can you move that to the documentation about `L.CRS.EPSG4326`? Also, try building the docs to see if it looks good.

**Comment by timrobertson100** [comment]
screenshot 2016-11-03 16 16 59

**Comment by IvanSanchez** [comment]
Please pay attention to [this bit in CONTRIBUTING.md](https://github.com/Leaflet/Leaflet/blob/master/CONTRIBUTING.md#improving-documentation):

> Since Leaflet 1.0.0-rc1, the API documentation in `reference-1.0.0.html` is handled
via [Leafdoc](https://github.com/Leaflet/Leafdoc). This means that next to the
code for every method, option or property there is a special code comment documenting
that feature. In order to edit the API documentation, just edit these comments in the
source code.

> On every release of a new Leaflet version, this file will be generated and copied over to docs/reference.html - there is no need to send pull requests with changes to this file to update the API documentation.

In other words: the changes should go on `src/geo/crs/CRS.EPSG4326.js`, not in `docs/reference-1.0.0.html`.

I guess I can copy-paste my words there.

**Comment by timrobertson100** [comment]
Sorry about that - and thanks.

**Comment by IvanSanchez** [comment]
No problem! I know it's a bit difficult to get it right on the first contribution to a project :-)
